examples/abitest/backend_module.sign

```diff
@@ -0,0 +1,12 @@
+-----BEGIN PUBLIC KEY-----

```

Comment:
Is this signature ever used? It seems that it is only the front-end module that requires downgrading permission.